.gitignore

```diff
@@ -7,8 +7,11 @@ typings/
 .coverage
 .mypy_cache
 .DS_Store
-.idea/
 .history/
 .pytest_cache/
 htmlcov/
-typings/
+
+.ipynb_checkpoints/
+.vscode/
```

Comment:
I think `.vscode` should be tracked because this is where we put useful settings for our VS Code users

Reply:
Reading this through this thread give me different perspectives. [Should I commit the .vscode folder to source control?](https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control)

I can exclude it.

Reply:
What in that thread gives you a different perspective? I read [the top answer](https://stackoverflow.com/a/32979933/15641512) and that aligns pretty well with the purpose of this repository. We want to share our settings, hence we check them in.

Reply:
I mean different perspective from my original thoughts. Pardon my bad English. And now I can see reasons why people would like to keep .vscod/ in.